sql/session.go

```diff
@@ -175,6 +175,8 @@ func DefaultSessionConfig() map[string]TypedValue {
 		"ndbinfo_version":          TypedValue{Text, ""},
 		"sql_select_limit":         TypedValue{Int32, math.MaxInt32},
 		"transaction_isolation":    TypedValue{Text, "READ UNCOMMITTED"},
+		"version":                  TypedValue{Text, ""},
```

Comment:
Maybe we can add the real version here to maintain consistency.

Reply:
What version should we put? A fake mysql version? It might not be consistent with `VERSION()` though

Reply:
we cannot use the same param as VERSION() is using here?

Reply:
That's the thing, VERSION() doesn't even exist in plain go-mysql-server. It's registered in gitbase with a specific version.

Reply:
oh, right, I mix both. Sorry for the noise.